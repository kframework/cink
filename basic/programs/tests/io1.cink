int main() {
  int x, y;
  cout << "x" << ", y" << " = ";
  cin >> x >> y;
  cout << x << y << endl;
  return 0;
}
