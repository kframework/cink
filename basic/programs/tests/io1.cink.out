x, y = 45
<T>
    <in>
        ListItem(#buffer ( "" ))
        ListItem(#istream ( 0 ))
    </in>
    <out>
        ListItem(#ostream ( 1 ))
        ListItem(#buffer ( "" ))
    </out>
    <control>
        <k>
            .K
        </k>
        <return>
            0
        </return>
        <fstack>
            .List
        </fstack>
        <sideEffects>
            .Bag
        </sideEffects>
        <env>
            main |-> #symScalLoc(1)
        </env>
    </control>
    <genv>
        main |-> #symScalLoc(1)
    </genv>
    <store>
        #symScalLoc(1) |-> lambda (  @ int x ; int y ; (((cout << "x") << 
          ", y") << " = ") ; ((cin >> x) >> y) ; (((cout << x) << y) << (endl)
          ) ; return 0 ; )
        #symScalLoc(2) |-> 4
        #symScalLoc(3) |-> 5
    </store>
</T>
